themes/digital.gov/layouts/shortcodes/cspan-iframe.html

```diff
@@ -0,0 +1,19 @@
+{{/*
+  Pass in the src url and the title attributes
+  {{< cspan-iframe src="https://www.c-span.org/video/standalone/?512480-1/confirmation-hearing-gsa-administration-intelligence-agencies-nominees" title="Confirmation Hearing for GSA Administration and Intelligence Agencies Nominees" >}}
+*/}}
+  
+{{- $src := .Get "src" -}}
+{{- $title := .Get "title" -}}
+
+<section class="cspan-embed">
```

Comment:
Consider adding aspect class and removing custom styles. I'd still keep the cspan-embed class for future use. So something like: `class="cspan-embed add-aspect-4x3"`.